src/mongoose/sanitizeQueryValue.ts

```diff
@@ -22,10 +22,7 @@ export const sanitizeQueryValue = ({ ctx, field, path, operator, val, hasCustomI
     if (!hasCustomID) {
       const isValid = mongoose.Types.ObjectId.isValid(val);
 
-      formattedValue = new mongoose.Types.ObjectId(val);
-
       if (!isValid) {
-        ctx.errors.push({ path });
```

Comment:
I'd like to talk through this change @jmikrut. If we are pushing errors here, then I think we need to do more formatting upfront. I think we go back to returning undefined and so the query can ignore it without any error.